### PR TITLE
Create self-healing.md

### PR DESCRIPTION
**Pull request by UjjawalTyagi123**
Signed-off-by: Ujjawal Tyagi <tyagiujjawal763@gmail.com>

Signed-off-by: Ujjawal Tyagi <91653886+UjjawalTyagi123@users.noreply.github.com>

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/pt-br/self-healing.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+status: Concluído
+category: propriedade
+tags: ["infraestrutura", "propriedade"]
+---
+
+Um sistema de autocorreção é capaz de se recuperar de certos tipos de falha sem qualquer intervenção humana.
+Possui um loop de "convergência" ou "controle" que analisa ativamente o estado real do sistema e
+compara-o com o estado inicialmente desejado pelos operadores.
+Se houver uma diferença (por exemplo, menos instâncias do aplicativo em execução do que o desejado),
+tomará uma ação corretiva (por exemplo, iniciar novas instâncias)
```
Comment (MrErlison) [inline content/pt-br/self-healing.md:12]: Minor: Add "."

File: content/pt-br/self-healing.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Autocura
+status: Concluído
+category: propriedade
+tags: ["infraestrutura", "propriedade"]
+---
+
+Um sistema de autocorreção é capaz de se recuperar de certos tipos de falha sem qualquer intervenção humana.
+Possui um loop de "convergência" ou "controle" que analisa ativamente o estado real do sistema e
```
Comment (MrErlison) [inline content/pt-br/self-healing.md:9]: Minor: "ciclo" instead of "loop"

File: content/pt-br/self-healing.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: Autocura
+status: Concluído
```
Comment (MrErlison) [inline content/pt-br/self-healing.md:3]: "Completed" instead of "Concluído". No need to translate.

File: content/pt-br/self-healing.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+title: Autocura
```
Comment (MrErlison) [inline content/pt-br/self-healing.md:2]: I like "autocorreção" in line 8. Update one of them to maintain consistency.

**Comment by MrErlison** [comment]
Hello @UjjawalTyagi123! Thanks for your contribution!

**Comment by edsoncelio** [comment]
Hey @UjjawalTyagi123 - If you need any help to do the requested changes, let us know :)

**Comment by edsoncelio** [comment]
Due to a lack of response, we are closing this PR.
Feel free to reopen at anytime @UjjawalTyagi123.

**Pull request closed without merging**